packages/xml-views-quick-fix/src/quick-fix-stable-id.ts

```diff
@@ -84,6 +92,7 @@ function computeQuickFixIdSuggestion(
   const suffix = biggestIdsByElementName[elementName]
     ? biggestIdsByElementName[elementName] + 1
     : 1;
+  biggestIdsByElementName[elementName] = suffix;
```

Comment:
document that we are mutating that data structure to support multiple fixes in the same file without conflicts